src/ethereum_clis/transition_tool.py

```diff
@@ -510,36 +516,21 @@ def evaluate(
         If a client's `t8n` tool varies from the default behavior, this method
         can be overridden.
         """
-        fork_name = fork.transition_tool_name(
-            block_number=env.number,
-            timestamp=env.timestamp,
-        )
-        if env.number == 0:
-            reward = -1
-        t8n_data = self.TransitionToolData(
-            alloc=alloc,
-            txs=txs,
-            env=env,
-            fork_name=fork_name,
-            chain_id=chain_id,
-            reward=reward,
-            blob_schedule=blob_schedule,
-            state_test=state_test,
-        )
-
         if self.t8n_use_server:
             if not self.process:
                 self.start_server()
             return self._evaluate_server(
-                t8n_data=t8n_data,
+                t8n_data=transition_tool_data,
                 debug_output_path=debug_output_path,
                 timeout=SLOW_REQUEST_TIMEOUT if slow_request else NORMAL_SERVER_TIMEOUT,
             )
 
         if self.t8n_use_stream:
-            return self._evaluate_stream(t8n_data=t8n_data, debug_output_path=debug_output_path)
+            return self._evaluate_stream(
```

Comment:
Does the EELS T8N  entry point wrapper get its own `_evaluate*` method or do we re-use `_evaluate_stream`?